src/main/resources/config/properties-form-runner.xml

```diff
@@ -240,6 +240,8 @@
     <property as="xs:string"  name="oxf.fr.pdf.font.path.ID"                                    value=""/>
     <property as="xs:string"  name="oxf.fr.pdf.font.family.ID"                                  value=""/>
     -->
+    <property as="xs:string"  name="oxf.fr.pdf.font.resource.roboto"                            value="/forms/resources/Roboto-Medium.ttf"/>
+    <property as="xs:string"  name="oxf.fr.pdf.font.family.roboto"                              value="Roboto"/>
     <property as="xs:string" name="oxf.fr.pdf.template.font.paths"                              value=""/>
 
```

Comment:
We will have to check if/why specifying an explicit font to embed is required. It's unclear to me why it's the case. Also I expect the behavior to be that this means some CSS refers to the `roboto` font, and I couldn't find an example. This just to say that we'll have to dig into this a little more.

Reply:
Yeah I was looking at this accessibility sample https://github.com/danfickle/openhtmltopdf/wiki/PDF-Accessibility-(PDF-UA,-WCAG,-Section-508)-Support where it has `useFont` and it says `Remember to add one or more fonts.` which makes me think this is necessary. However it turns out this is not the cause of failure. Will remove.